Modules/Sources/WordPressUI/Views/Acknowledgements/AcknowledgementsListView.swift

```diff
@@ -0,0 +1,54 @@
+import SwiftUI
+
+public struct AcknowledgementsListView: View {
+
+    @ObservedObject
+    var viewModel: AcknowledgementsListViewModel
+
+    public init(viewModel: AcknowledgementsListViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        Group {
+            if let error = viewModel.error {
+                EmptyStateView(error.localizedDescription, systemImage: "exclamationmark.triangle.fill")
+                    .multilineTextAlignment(.center)
+            } else if viewModel.isLoadingItems {
+                ProgressView()
+            } else {
+                List(viewModel.items) { item in
+                    NavigationLink {
+                        AcknowledgementsDetailView(item: item)
+                    } label: {
+                        VStack(alignment: .leading) {
+                            Text(item.title)
+                                .font(.body)
+                            if let description = item.description {
+                                Text(description)
+                                    .font(.caption)
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        .onAppear(perform: viewModel.onAppear)
```

Comment:
(nit) Instead of `onAppear` and manually managing `taskHandle: Task<Void, Error>?`, you can use the native [`task`](https://developer.apple.com/documentation/swiftui/view/task(priority:_:)) modifier.

Reply:
Tried that in feb005a028c3d80d3efe9fd7bed5a9b44fec15f2 – I think I still want all the logic in the view model but it cleans up a few lines so probably a good trade even if I can't pass the function as a reference

Reply:
If you call it viewModel.onAppear and make it async, you can feel better about not having logic in the view 😉